bigframes/operations/aggregations.py

```diff
@@ -42,6 +42,11 @@ def uses_total_row_ordering(self):
     def can_order_by(self):
         return False
 
+    @property
+    def order_independent(self):
+        """Whether the output of the operator depends on the ordering of input rows. Navigation functions are a notable case."""
```

Comment:
Let's keep the title line short.

```suggestion
        """True if the output of the operator depends on the ordering of input rows.

        Navigation functions are a notable case.
        """
```

Reply:
done